pkg/operator/collection.go

```diff
@@ -239,9 +239,10 @@ func (r *collectionReconciler) ensureCollectorDaemonSet(ctx context.Context, spe
 		for _, ev := range c.Env {
 			if ev.Name != "EXTRA_ARGS" {
 				repl = append(repl, ev)
+			} else {
+				repl = append(repl, corev1.EnvVar{Name: "EXTRA_ARGS", Value: strings.Join(flags, " ")})
```

Comment:
what if the `EXTRA_ARGS` flag doesn't exist yet? then none of these will get set

Reply:
Redacted this change as this was incorrect.